sdk/src/main/java/com/bugsnag/android/Configuration.java

```diff
@@ -641,6 +645,58 @@ public void setDelivery(@NonNull Delivery delivery) {
         this.delivery = delivery;
     }
 
+    /**
+     * @return whether ANRs will be captured or not
+     * @see #setDetectAnrs(boolean)
+     */
+    public boolean getDetectAnrs() {
+        return detectAnrs;
+    }
+
+    /**
+     * Sets whether <a href="https://developer.android.com/topic/performance/vitals/anr">ANRs</a>
+     * should be reported to Bugsnag. By default, Bugsnag will record an ANR whenever the main
+     * thread has been blocked for 5000 milliseconds or longer, when no debugger is attached to
+     * the app.
+     * <p/>
+     * If you wish to disable ANR detection, you should set this property to false; if you wish to
+     * configure the time threshold required to capture an ANR, you should use the
+     * {@link #setAnrThresholdMs(long)} property.
+     *
+     * @param detectAnrs whether ANRs should be captured or not
+     * @see #setAnrThresholdMs(long)
+     */
+    public void setDetectAnrs(boolean detectAnrs) {
+        this.detectAnrs = detectAnrs;
+    }
+
+    /**
+     * @return the threshold at which ANRs are detected, in ms
+     * @see #setAnrThresholdMs(long)
+     */
+    public long getAnrThresholdMs() {
+        return anrThresholdMs;
+    }
+
+    /**
+     * Sets the time in milliseconds at which an
+     * <a href="https://developer.android.com/topic/performance/vitals/anr">ANR</a> is detected
+     * by Bugsnag. By default, Bugsnag will record an ANR whenever the main thread has been blocked
+     * for 5000 milliseconds or longer.
+     * <p/>
+     * If you wish to disable ANR detection completely, you should set the
+     * {@link #setDetectAnrs(boolean)} property to false.
+     * <p/>
+     * Attempting to set this property to any value below 100ms will result in the anrThresholdMs
```

Comment:
why do you want to adjust this value if an invalid one is supploed, rather than throw an `IllegalArgumentException`?

Reply:
@duncanhewett did you have any opinions on this one? I'd be happy with either setting the value to 100ms, or throwing an exception immediately.

Reply:
I think if it's less than 1000ms it will throw an `IllegalArgumentException` here anyway https://github.com/bugsnag/bugsnag-android/blob/android-anr-detection/sdk/src/main/java/com/bugsnag/android/BlockedThreadDetector.java#L49

Reply:
Either is fine with me.

Reply:
In that case I will leave the current implementation, which sets the anrThresholdMs as 100ms.